File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Ipify SDK exists test

require_once __DIR__ . '/../ipify_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = IpifySDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Ipify SDK exists test

require "minitest/autorun"
require_relative "../Ipify_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = IpifySDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
